### ISSUE TITLE
[REMOVE] Toki

### ISSUE THREAD

**Issue by onlyjob** [tycrek/degoogle#100]
__Toki__ was listed by mistake, incorrectly described. Please remove it.

Quite obviously _Toki.com_ is an unlisted instance of _Searx_ which is _not_ distributed but a _meta_ search engine. From a quick glance it have numerous problems such as lack of source (e.g. public code repository, issues management system, etc.), UI customisations to remove references to Searx (commented in HTML source), etc.

Basically Toki is a dodgy service that is not qualified (unworthy) to be listed here.

**Comment by tycrek** [comment]
Comparing [Toki](https://toki.com/) to other Searx instances such as https://searx.be/ and https://searx.info/, it is quite clear it is a clone with UI customizations. Even without considering the privacy ramifications, this is blatantly stealing the Searx code without credit. I'll remove it from the list, thanks.
